roles/linkwarden/defaults/main.yml

```diff
@@ -0,0 +1,146 @@
+##########################################################################
+# Title:         Sandbox: LinkWarden | Default Variables                 #
+# Author(s):     CHAIR/Raneydazed                                        #
+# URL:           https://github.com/saltyorg/Sandbox                     #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+linkwarden_name: linkwarden
+
+################################
+# Paths
+################################
+
+linkwarden_paths_folder: "{{ linkwarden_name }}"
+linkwarden_paths_location: "{{ server_appdata_path }}/{{ linkwarden_paths_folder }}"
+linkwarden_paths_folders_list:
+  - "{{ linkwarden_paths_location }}"
+
+################################
+# Web
+################################
+
+linkwarden_web_subdomain: "{{ linkwarden_name }}"
+linkwarden_web_domain: "{{ user.domain }}"
+linkwarden_web_port: "3000"
+linkwarden_web_url: "{{ 'https://' + (linkwarden_web_subdomain + '.' + linkwarden_web_domain
```

Comment:
Start parenthesis with no mate in the correct side of the if/else

Reply:
I'm afraid I don't understand, as I'd taken this from a role you'd made. I don't remember which, but I just compared it to joplin and its the same. Is this how you want it? 
```yaml
linkwarden_web_url: "{{ 'https://' + (linkwarden_web_subdomain + '.' + linkwarden_web_domain
                     if linkwarden_web_subdomain | length > 0
                     else linkwarden_web_domain) }}"
```

Reply:
## Joplin
```yaml
...
joplin_web_url: "{{ 'https://' + (joplin_web_subdomain + '.' + joplin_web_domain
                 if (joplin_web_subdomain | length > 0)
                 else joplin_web_domain) }}"
...
```

## Codex because why not

```yaml
codex_web_url: "{{ 'https://' + (codex_web_subdomain + '.' + codex_web_domain
                if (codex_web_subdomain | length > 0)
                else codex_web_domain) }}"
```

Reply:
You are correct, I was doing something else while looking at this and clearly misread it. My bad.